rmgpy/molecule/group.py

```diff
@@ -1368,20 +1368,22 @@ def sortByConnectivity(atomList):
             Returns: a sorted list of atoms where each atom is connected to a previous
```

Comment:
Can you add a few unittests for just this method `sortByConnectivity` rather than other higher level methods in `groupTest.py`; also choose at least examples that fall into different cases, e.g. one can be whole molecule is connected by `B` bond, the other can be divided into several sub-molecules.